### ISSUE TITLE
[Bug] "Originator Telemetry" rule node not calculating the right SUM

### ISSUE THREAD

**Issue by apadEnlace** [thingsboard/thingsboard#6694]
**Describe the bug**

I have the following set up:  "Originator Telemetry" rule nodes in parallel, one calculating the "SUM" and the other one giving "ALL" the values. This is the set up:

![image](https://user-images.githubusercontent.com/99991163/173165179-cdaef061-d664-4ba3-ad09-4b7adffb85af.png)

![image](https://user-images.githubusercontent.com/99991163/173165207-f05b1d1e-f94d-48b4-85c0-c596f735a1ea.png)

![image](https://user-images.githubusercontent.com/99991163/173165217-6d7faf2b-fa1f-4b2b-a4a5-2e15f6cdb581.png)

This is the data that the ALL rule node is seeing:

![image](https://user-images.githubusercontent.com/99991163/173164685-90503702-f30e-4306-8f9a-6a19b5e7855c.png)

This the text in the above picture:

```
{
    "Battsys_Ebat_MPSinUse": "[{\"ts\":1654632683535,\"value\":0.190439},{\"ts\":1654632599311,\"value\":1.717056},{\"ts\":1654632425327,\"value\":\"0.19112900000000002\"},{\"ts\":1654632359441,\"value\":\"0.47730499999999987\"},{\"ts\":1654631972506,\"value\":\"0.09539199999999999\"},{\"ts\":1654631880869,\"value\":\"0.09573699999999999\"},{\"ts\":1654631422171,\"value\":0.047351},{\"ts\":1654631270195,\"value\":0.152276},{\"ts\":1654628828891,\"value\":0},{\"ts\":1654627150760,\"value\":0.620048},{\"ts\":1654627058379,\"value\":\"0.09573699999999999\"},{\"ts\":1654624501278,\"value\":\"0.010004999999999997\"},{\"ts\":1654616504601,\"value\":3.45E-4}]",
    "Duration_in_Hrs": "[{\"ts\":1654632683535,\"value\":0.02},{\"ts\":1654632599311,\"value\":0.05},{\"ts\":1654632425327,\"value\":0.02},{\"ts\":1654632359441,\"value\":0.05},{\"ts\":1654631972506,\"value\":0.01},{\"ts\":1654631880869,\"value\":0.02},{\"ts\":1654631422171,\"value\":0.02},{\"ts\":1654631270195,\"value\":0.3},{\"ts\":1654628828891,\"value\":0},{\"ts\":1654627150760,\"value\":0.03},{\"ts\":1654627058379,\"value\":0.71},{\"ts\":1654624501278,\"value\":0.52},{\"ts\":1654616504601,\"value\":20.16}]",
    "scope": "SERVER_SCOPE",
    "ss_coreReportingEndTime": "1654660800000",
    "ss_coreReportingStartTime": "1654574400000",
    "userId": "bb6be690-334a-11ec-b69b-df55d593f7d9",
    "userName": "-"
}
```

This is the data in the "SUM" node:

![image](https://user-images.githubusercontent.com/99991163/173164787-9949e0ee-cecd-47e4-8f15-817dd181b665.png)

This is the text of the above screen:

`{
    "Battsys_Ebat_MPSinUse": "[{\"ts\":1654617600000,\"value\":**2.727515**}]",
    "Duration_in_Hrs": "[{\"ts\":1654617600000,\"value\":21.910000000000004}]",
    "scope": "SERVER_SCOPE",
    "ss_coreReportingEndTime": "1654660800000",
    "ss_coreReportingStartTime": "1654574400000",
    "userId": "bb6be690-334a-11ec-b69b-df55d593f7d9",
    "userName": "-"
}

The total "2.727515" is not correct. When I put all the individual values in the time frame in Excel, the total should be "3.69282":

![image](https://user-images.githubusercontent.com/99991163/173164991-66bc090c-1a3c-4a41-9712-86147a7318ee.png)

**What am I missing? In the above screenshots, you can see that the Start and End times are the same in both nodes, as well as the telemetry I'm reading from (Battsys_Ebat_MPSinUse)**

Thanks in advanced for your help!

**Your Server Environment**
* own setup
  * local infrastructure
  * ThingsBoard v.3.3.4.1PE
  * OS: Ubuntu 20.04

**Your Client Environment**
* OS: Windows 10
* Browser: Chrome Version 100.0.4896.60

**Comment by sumin-denys** [comment]
Hi @apadEnlace,

Thank you for your report! We know about this issue and have PROD-1691 already in our backlog!

I am closing this issue for now. If you need to update the status of the prod ticket, please feel free to comment on this issue.

Please refer to our release notes to find out more about new features and bug fixes: [Release Notes](https://thingsboard.io/docs/pe/reference/releases/).
If you have any further questions, please feel free to create a new issue or reopen this one.
